sources/io/tests/streams.dylan

```diff
@@ -17,225 +17,188 @@ end;
 
 /// Stream convenience function tests
 
-register-stream-test(<stream>, test-read-character, direction: #"input");
-register-stream-test(<stream>, test-read-text, direction: #"input");
-register-stream-test(<stream>, test-read-text-into!, direction: #"input");
-register-stream-test(<stream>, test-read-line, direction: #"input");
-register-stream-test(<stream>, test-read-line-into!, direction: #"input");
-register-stream-test(<stream>, test-read-to, direction: #"input");
-register-stream-test(<stream>, test-read-through, direction: #"input");
-register-stream-test(<stream>, test-read-to-end, direction: #"input");
-register-stream-test(<stream>, test-skip-through, direction: #"input");
-register-stream-test(<stream>, test-write-text, direction: #"output");
-register-stream-test(<stream>, test-write-line, direction: #"output");
-register-stream-test(<stream>, test-new-line, direction: #"output");
-
-// Don't test the functions we're already testing... there must be a better way!
-define streams function-test read-character () end;
-define streams function-test read-text () end;
-define streams function-test read-text-into! () end;
-define streams function-test read-line () end;
-define streams function-test read-line-into! () end;
-define streams function-test read-to () end;
-define streams function-test read-through () end;
-define streams function-test read-to-end () end;
-define streams function-test skip-through () end;
-define streams function-test write-text () end;
-define streams function-test write-line () end;
-define streams function-test new-line () end;
-
-define method test-read-character
+register-stream-test(<stream>, %test-read-character, direction: #"input");
+register-stream-test(<stream>, %test-read-text, direction: #"input");
+register-stream-test(<stream>, %test-read-text-into!, direction: #"input");
+register-stream-test(<stream>, %test-read-line, direction: #"input");
+register-stream-test(<stream>, %test-read-line-into!, direction: #"input");
+register-stream-test(<stream>, %test-read-to, direction: #"input");
+register-stream-test(<stream>, %test-read-through, direction: #"input");
+register-stream-test(<stream>, %test-read-to-end, direction: #"input");
+register-stream-test(<stream>, %test-skip-through, direction: #"input");
+register-stream-test(<stream>, %test-write-text, direction: #"output");
+register-stream-test(<stream>, %test-write-line, direction: #"output");
+register-stream-test(<stream>, %test-new-line, direction: #"output");
+
+define method %test-read-character
     (info :: <stream-test-info>, stream :: <stream>) => ()
   //---*** Fill this in...
-end method test-read-character;
+end method;
 
-define method test-read-text
+define method %test-read-text
     (info :: <stream-test-info>, stream :: <stream>) => ()
   //---*** Fill this in...
-end method test-read-text;
+end method;
 
-define method test-read-text-into!
+define method %test-read-text-into!
```

Comment:
Could we consider calling these `do-test-read-text-into!` etc.? Using `%` would seem to have different implications.

Reply:
Sure, will do.

Reply:
done